### ISSUE TITLE
[ACTIVE_ARBITRATION_WIN] SWARM-1786 — ArXiv+LangUK post-admission global-dedup successor

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1786]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1786
STATUS: ACTIVE_STALE_TAKEOVER_ARBITRATION_WIN
SWARM_LANE_KEY: D03|ARXIV-LANGUK|POST-ADMISSION-GLOBAL-DEDUP-INTAKE|INCUMBENT-V9-DELEGATION-V1
SELECTED_WORK_KIND: STALE TAKEOVER / SAME-SEMANTIC PRODUCT RESTORATION / DATA AUTHORITY CONVERGENCE
LOCAL_FREE_ONLY: true
MERGE_AUTHORITY: false
LIVE_MAIN_AT_WIN: 2da11c62f066671a1122888ec116242502f4fb9c
CLAIMED_AT_UTC: 2026-09-14T00:32:34Z
LEASE_UNTIL_UTC: 2026-09-14T06:32:34Z unless terminal/released sooner

POSTCLAIM DIRECT ARBITRATION — WIN:
- direct OPEN issues newest-first at 100/page: pages 1..12 populated, page 13 exact `[]` sentinel;
- direct OPEN PRs newest-first at 100/page: pages 1..5 populated, page 6 exact `[]` sentinel;
- race refresh of exact semantic returns #1786 plus stale predecessors #1566/#1530/#1397 only; no newer same-semantic owner exists;
- exact semantic OPEN PR search returns none;
- main race-refresh remains exactly `2da11c62f066671a1122888ec116242502f4fb9c`;
- newer PR #1787 is the disjoint Rada-laws Q/P adapter semantic and does not own this lane.

PREDECESSOR / TAKEOVER BASIS:
- #1397 originally created PR #1404.
- independent audit #1403 rejected old head `a0a6060c7fb0bb6358f785b5650050117a6a8fe0` with D03-INTAKE-001 HIGH: caller-controlled incumbent inventory/payload graph was accepted under only a correct #1055 SHA label.
- #1530 lease expired 2026-09-12T13:38:15Z; #1566 is stale and its own later #1580 correction established that PR #1404 had been reset/closed.
- PR #1404 is CLOSED / NOT MERGED with head==base and zero Product delta. Therefore there is no live branch Product to mutate in place; restoring this abandoned exact semantic requires one current-main successor lineage, not revival of stale bytes or a competing architecture.

OWNED PRODUCT SURFACE:
- one successor branch/PR from exact live main for the abandoned #1404 semantic;
- reuse exact ArXiv/LangUK admission authentication from rejected lineage where still valid;
- remove the rejected caller-supplied incumbent-base trust model;
- first prove current incumbent V9 authority via `expanded_global_dedup_v9.run_expanded_dedup(...)` over authenticated V8+Rada inputs, reconstruct exact base+Rada matcher inputs with incumbent helpers, cross-check against V9 source vector, then append only authenticated ArXiv+LangUK candidate rows and invoke the same incumbent matcher callbacks;
- add coherent substituted/resealed incumbent-base adversarial regression;
- preserve zero downstream/corpus/training credit until real expanded matcher execution and all downstream gates.

AVOID:
No matcher-science changes; no mutation of #1345, #1787, balance/split/packing/tokenizer/model/trainer/recovery/eval/final-test lineages; no paid compute; no foreign pretrained weights; no external LLM/API corpus intelligence.

TRUTH_BOUNDARY_AT_WIN:
ARXIV_LANGUK_ALREADY_REAL_ADMITTED_PREDEDUP_BYTES=3949184
THIS_GLOBAL_DEDUP_EXECUTED=false
CANONICAL_CAPACITY_CREDITED=0
TRAINING_AUTHORIZED_BYTES=0
AUTHORIZED_UNIQUE_LOSS_POSITIONS=0
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false
